types/zipkin-javascript-opentracing/zipkin-javascript-opentracing-tests.ts

```diff
@@ -0,0 +1,9 @@
+import * as ZipkinOpentracing from "zipkin-javascript-opentracing";
```

Comment:
Since `ZipkinOpentracing` is a class, that is not supported by ES6 module, it should be `import ZipkinOpentracing 
 = require("zipkin-javascript-opentracing");` here.

Reply:
Yep you're right, done inside the last commit ;-)